contributions/api/controllers/configs.py

```diff
@@ -26,6 +26,7 @@
 API_LOC = os.getenv('API_LOC', '../')
 CONTRIBUTION_URL_PREFIX = os.getenv('CONTRIBUTION_URL_PREFIX', '')
 DEBUG = bool(os.getenv('DEBUG', 'False') == 'True')
+CORS_ENABLED = bool(os.getenv('CORS_ENABLE', 'False') == 'True')
```

Comment:
```suggestion
CORS_ENABLED = bool(os.getenv('CORS_ENABLED', 'False') == 'True')
```
Suggesting name change for consistency.